src/main/java/seedu/tp/commands/GroupCommand.java

```diff
@@ -24,7 +24,9 @@ public GroupCommand(GroupFactory groupFactory, GroupList groupList) {
 
     @Override
     public void execute() throws InvalidFlashcardIndexException {
+        LOGGER.info("Creating a new group");
```

Comment:
Consider adding `...` to the end of the string to be consistent with other log messages